### DISCUSSION TITLE
[Kavita+] Add Support for Ryot Scrobbling

### DISCUSSION THREAD

**Discussion by b3nd1k** [Kareadita/Kavita#3464] [Ideas]
### Idea Description

One of the standout features of Kavita+ is the ability to scrobble reading data to third-party websites. I would love to see [Ryot](https://ryot.io) added as a supported platform. Similar to AniList and MAL, Ryot provides an API key that users can generate and integrate with Kavita.

Adding support for Ryot would enhance the experience for users who actively track their reading across multiple platforms and streamline their workflows. Thank you for considering this suggestion!

### Idea Category

Feature Enhancement

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by IgnisDa** [comment]
Ryot author here. Since Kavita already contains information about the Anilist ID, the data can be sent to Ryot in a webhook endpoint. Since both projects already have scrobbling support from Anilist, this should not be very difficult to implement.

**Comment by majora2007** [comment]
I had already reached out to IgnisDa about integration awhile ago, but from the issue it looked like all media needed ISBNs to be trackable, which is not really do-able. Has this changed @IgnisDa ? 

https://github.com/IgnisDa/ryot/issues/196

**Reply by IgnisDa** [reply]
Yes. THat is no longer the case. Ryot can now work with Anilist list IDs.
